test/Interop/Cxx/templates/Inputs/transitive-function-template.h

```diff
@@ -0,0 +1,54 @@
+#ifndef TEST_INTEROP_CXX_TEMPLATES_INPUTS_TRANSITIVE_FUNCTION_TEMPLATE_H
+#define TEST_INTEROP_CXX_TEMPLATES_INPUTS_TRANSITIVE_FUNCTION_TEMPLATE_H
+
+template<typename T>
+inline T increment(T t) {
+  return t + 1;
+}
+
+template<typename T>
+inline T add2(T t) {
+  return increment(increment(t));
+}
+
+template<typename T>
+inline T add3(T t) {
+  return increment(add2(t));
+}
+
+// template<typename T>
```

Comment:
If these are going to remain commented out can you add a comment explaining? Or maybe a link to an SR?

Reply:
This was unintentionally added to this PR, I removed it.